src/components/custom/platforms-dropdown.js

```diff
@@ -68,13 +73,27 @@ const getNavigationContents = (parent, is_ppc, is_ppc_redirect) => {
     if (parent === 'resources') return <NavResources />
 }
 
-const PlatformsDropdown = ({ current_ref, is_ppc, is_ppc_redirect, parent, setActiveDropdown }) => {
+const PlatformsDropdown = ({
+    current_ref,
+    is_ppc,
+    is_ppc_redirect,
+    parent,
+    setActiveDropdown,
+    active_dropdown,
+}) => {
+    const is_trade = active_dropdown === 'trade'
+    const setTradeArrowOffset = () =>
+        current_ref.offsetLeft - (window.innerWidth * 0.1) / 2 + current_ref.offsetWidth / 2 - 15
     const [left_offset, setLeftOffset] = useState(current_ref.offsetLeft)
-    const [left_arrow_offset, setLeftArrowOffset] = useState(current_ref.offsetWidth / 2 - 15)
+    const [left_arrow_offset, setLeftArrowOffset] = useState(
+        is_trade ? setTradeArrowOffset() : current_ref.offsetWidth / 2 - 15,
+    )
     const dropdownContainerRef = useRef(null)
 
     const updateOffsets = useCallback(() => {
-        if (current_ref) {
+        if (is_trade) {
+            setLeftArrowOffset(setTradeArrowOffset())
+        } else if (current_ref && !is_trade) {
             setLeftOffset(current_ref.offsetLeft)
             setLeftArrowOffset(current_ref.offsetWidth / 2 - 15)
         }
```

Comment:
Could you add a comment here to explain the logic for the calculation?

Reply:
In the case of trade dropdown we need to change view of it. So the rules of offset calculation is different. I use `is_trade` variable to detect selected tab. `setTradeArrowOffset` function counts place where arrow suppose to be (left offset - half of left flex margin - half of element offset with little adjust in 15px)